tests/test_level_04.py

```diff
@@ -407,3 +407,46 @@ def test_chained_assignments(self):
       code=code,
       expected=expected
     )
+
+
+  def test_meta_column_missing_closing_quote(self):
+    code = textwrap.dedent("""\
+    print 'Hello'
+    print 'World""")
+
+    instance = hedy.IsValid()
+    instance.level = self.level
+    program_root = hedy.parse_input(code, self.level, 'en')
+    is_valid = instance.transform(program_root)
+    _, invalid_info = is_valid
+
+    invalid_info = invalid_info[0]
+
+    line = invalid_info.line
+    column = invalid_info.column
+
+    # Boryana Jan 22
+    # Proabably we want to change the column so that it points to
```

Comment:
I was thinking of doing this check in the repair itself, since we know what and where the error is.